crates/oxc_ast/src/ast/js.rs

```diff
@@ -1695,9 +1699,11 @@ pub enum FormalParameterKind {
 #[ast(visit)]
 #[derive(Debug)]
 #[generate_derive(CloneIn, GetSpan, GetSpanMut, ContentEq, ESTree)]
+#[estree(rename = "BlockStatement")]
 pub struct FunctionBody<'a> {
     pub span: Span,
     pub directives: Vec<'a, Directive<'a>>,
+    #[estree(rename = "body")]
     pub statements: Vec<'a, Statement<'a>>,
```

Comment:
both `directives` and `statements` should be in `body`, but `append_to` macro seems to break for non optional field. I'll look into updating macro for this case separately.

Reply:
Yes, extending `#[estree(append_to = ...)]` to cover appending `Vec`s too would be ideal. We could alternatively add `#[estree(prepend_to = ...)]`, which would make more sense here. But I think for now we already have enough attributes - probably better to use `append_to` at least for now.

Let me know if you have trouble working out how to adapt the codegen. I think it *mostly* makes sense, but I wrote it, so of course it makes sense *to me*!